.github/workflows/build-and-test.yml

```diff
@@ -1425,7 +1435,7 @@ jobs:
   # macOS jobs
   swig_macOS:
     name: "Mac|Build SWIG"
-    runs-on: macos-10.15
+    runs-on: macos-12
```

Comment:
that's unrelated to the python work, please rebase ?

Reply:
Yep the correct PR is https://github.com/coqui-ai/STT/issues/2278

Reply:
Fixed